gcp-test/tests/gcp_compute_target_vpn_gateway/variables.tf

```diff
@@ -36,7 +36,10 @@ data "null_data_source" "resource" {
 }
 
 resource "google_compute_network" "network_test" {
-  name = var.resource_name
+  name                    = var.resource_name
```

Comment:
Why can't we use the basic network configuration for running our test?
```
resource "google_compute_network" "vpc_network" {
  name = "vpc-network"
}
```

Reply:
It is failing some times If there is a resource already available with same name so I made the resource name dynamic

Reply:
In `1.8.0` it was failed due to resource was already, exists so I made it dynamic in `1.8.1-beta.0`